### PR TITLE
Remove all deprecated resources for `v2.0`

### LINKED ISSUE #2761: Deprecated resources to be removed for `v2.0`
- [ ] `aiida.load_dbenv`
- [ ] `aiida.try_dbenv`
- [ ] `aiida.is_dbenv_loaded`
- [ ] `aiida.calculations.plugins`
- [ ] `-t/--type` option of `verdi group list`
- [ ] `verdi devel tests`
- [ ] `verdi node tree`
- [ ] Tree drawers in `aiida.cmdline.utils.ascii_vis`
- [ ] `aiida.common.datastructures.CalcInfo.retrieve_singlefile_list`
- [ ] `aiida.manage.external.pgsu`
- [ ] `aiida.manage.tests.unittest_classes`
- [ ] `Autogroup.get_group_name` and `Autogroup.set_group_name`
- [ ] `Computer.get_schema`
- [ ] `Group.get_schema`
- [ ] `aiida.orm.groups.GroupTypeString`
- [ ] `raise_error` argument of `Code.relabel`
- [ ] `filename` guard in `SinglefileData`
- [ ] `encoding` and `mode` of `Node.put_object_from_file`

### PR DESCRIPTION
**Pull request by sphuber**
Fixes #2761

### CODE REVIEW

File: .github/workflows/tests.sh [hunk 1]
```diff
@@ -29,7 +29,6 @@ verdi daemon stop
 pytest --noconftest ${SYSTEM_TESTS}/test_test_manager.py
 pytest --noconftest ${SYSTEM_TESTS}/test_ipython_magics.py
 pytest --noconftest ${SYSTEM_TESTS}/test_profile_manager.py
-python ${SYSTEM_TESTS}/test_plugin_testcase.py  # uses custom unittest test runner
```
Comment (chrisjsewell) [inline .github/workflows/tests.sh:32]: This isn't technically a deprecation is it?
I'm personally not too bothered if you remove it, but what was the reasoning for doing so?

Comment (sphuber) [reply .github/workflows/tests.sh:32]: If I understood correctly, the unittest test case manager has been deprecated and this was just testing it. So no more need for it right?

**Review by chrisjsewell — APPROVED** [review]
one quick question, but yeh all good ta

File: .github/workflows/tests.sh (hunk 1 above)
Comment (sphuber) [reply .github/workflows/tests.sh:32]: It was calling the `TestRunner` and `PluginTestCase` from `aiida.manage.tests.unittest_classes` which included a deprecation warning for v2.0 saying it was going to be removed. I am pretty sure we only support `pytest` from now on

Comment (chrisjsewell) [reply .github/workflows/tests.sh:32]: yeh actually thats fine

File: aiida/tools/importexport/dbexport/__init__.py [hunk 2]
```diff
@@ -87,18 +81,6 @@ def export(
         set_progress_bar_tqdm(leave=True)
         export(...)
 
-    .. deprecated:: 1.5.0
-        Support for the parameter `silent` will be removed in `v2.0.0`.
-        Please set the log level and progress bar implementation independently.
-
-    .. deprecated:: 1.5.0
```
Comment (chrisjsewell) [inline aiida/tools/importexport/dbexport/__init__.py:94]: here

**Pull request merged**